src/common/components/ResultItem.jsx

```diff
@@ -1,39 +0,0 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import { List } from 'antd';
-
-import './ResultItem.scss';
-
-class ResultItem extends Component {
```

Comment:
are you sure we don't need this kinda component, where we have common stuff for all result items, no matter what?

actually what's the replacement thing for this?